<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClienteDialog</class>
 <widget class="QDialog" name="ClienteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar Cliente</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupDatosCliente">
     <property name="title">
      <string>Datos del Cliente</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNombre">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineNombre">
        <property name="placeholderText">
         <string>Ingrese el nombre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelApellidoPaterno">
        <property name="text">
         <string>Apellido Paterno:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineApellidoPaterno">
        <property name="placeholderText">
         <string>Ingrese el apellido paterno</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEmail">
        <property name="placeholderText">
         <string>Ingrese el email</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBotones">
     <item>
      <widget class="QPushButton" name="btnGuardarCliente">
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 0); color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelarCliente">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 0, 0); color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
